Singularity/Singularity/Screen/ScreenClasses/AchievementsScreen.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace Singularity.Screen.ScreenClasses
+{
+    /// <inheritdoc cref="IScreen"/>
+    /// <summary>
+    /// Used to show achievements that the player has earned.
+    /// It shows achievements already earned in a list format, not
+    /// unsimilar to how steam shows achievements. It has no buttons
+    /// other than back and simply shows textures of the achievements
+    /// and their description. If an achievement has not been earned yet,
+    /// the achievement texture is blacked out.
+    /// </summary>
+    class AchievementsScreen : IScreen
+    {
+        /// <summary>
+        /// Updates the contents of the screen.
+        /// </summary>
+        /// <param name="gametime">Current gametime. Used for actions
+        /// that take place over time </param>
+        public void Update(GameTime gametime)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Draws the content of this screen.
+        /// </summary>
+        /// <param name="spriteBatch">spriteBatch that this object should draw to.</param>
+        public void Draw(SpriteBatch spriteBatch)
+        {
+            throw new NotImplementedException();
```

Comment:
maybe here even?

Reply:
It's just currently not implemented because it's not yet implemented. It'll be implemented later once we figure out what we want on the achievements screen and the assets have been made

Reply:
ah alright.